### ISSUE TITLE
Unable to run on Google Colaboratory

### ISSUE THREAD

**Issue by Silviovespoli** [t-wolfeadam/Alpyne#10]
Hello, 
I'm trying to implement the Alpyne on Google Colaboratory (which may be useful for allowing the development of reinforcement learning widespread). Locally your library works well, while on Google Colaboratory the algorithm stuck on this particular line:
`    client = AlpyneClient(r"/model.jar", blocking=True, verbose=True)`
and in particular, stopping the execution, the problem seems to be here in the define of the Alpyne Client class:

```
try:
            self._http_client = HttpClient(f"http://127.0.0.1:{port}")
            _, _, output = self._http_client.get("/versions/number/0")
            self.version = ModelVersion.from_json(output)
except:            
            raise ModelError(f"Failed to properly start the app. Error from process: {self._proc.stderr.readline().decode()}")
```

I tried to propose a different port, but nothing worked at the moment. Have you any suggestions about this point?

**Comment by t-wolfeadam** [comment]
Interesting use case! I don't know anything about how Google Colaboratory hosts files or what networking permissions it does or does not allow. To try to see if this is a solvable, I'll first need the alpyne.log file (with `verbose=True`).

**Comment by Silviovespoli** [comment]
Hello, thanks for your answer!
So, this is the content of the log file that I receive from the execution of the model.
```
[2021-12-01 19:39:50] INFO    @ com.anylogic.alpyne.AlpyneServer main 
                              | [PID=299] Initializing @ '/content/drive/MyDrive/Anylogic/Traffic Light RL' with arguments: {path=., port=51150, dev_mode=false}
[2021-12-01 19:39:50] FINER   @ ModelManager.<init>
                              | ENTRY {0}	.
[2021-12-01 19:39:50] FINER   @ BonsaiBrainLearningPlatform.<init>
                              | ENTRY {0}	{}
[2021-12-01 19:39:51] INFO    @ Version.load
                              | Loading from experiment
[2021-12-01 19:39:51] INFO    @ Version.load
                              | Inputs      = [(DOUBLE)timeNS=null, (DOUBLE)timeEW=null, (BOOLEAN)doRLAction=null, (DOUBLE)secsBetweenActions=null, (STRING)scheduleNameNS=null, (STRING)scheduleNameEW=null, (DOUBLE){START_TIME}=0.0, (DATE_TIME){START_DATE}=Tue Jan 01 09:00:00 UTC 2019, (DOUBLE){STOP_TIME}=480.0, (DATE_TIME){STOP_DATE}=Tue Jan 01 17:00:00 UTC 2019, (LONG){RANDOM_SEED}=null]
                                Outputs     = [(DATA_SET)EWPhaseLenDS=null, (DATA_SET)NSPhaseLenDS=null, (DATA_SET)EWCountDS=null, (DATA_SET)NSCountDS=null, (DATA_SET)tisDS=null, (DATA_SET)tisDS_NS=null, (DATA_SET)tisDS_EW=null, (HISTOGRAM_DATA)speedsNS=null, (HISTOGRAM_DATA)speedsEW=null, (DATA_SET)previewNS=null, (DATA_SET)previewEW=null, (INTEGER)totalPassed=null, (DOUBLE)meanTIS=null, (DOUBLE)maxTIS=null]
                                Config      = [(DOUBLE)secs_per_phase=null, (DOUBLE)secs_between_actions=null, (STRING)schedule_ns=null, (STRING)schedule_ew=null, (DOUBLE){START_TIME}=0.0, (DATE_TIME){START_DATE}=Tue Jan 01 09:00:00 UTC 2019, (DOUBLE){STOP_TIME}=480.0, (DATE_TIME){STOP_DATE}=Tue Jan 01 17:00:00 UTC 2019, (LONG){RANDOM_SEED}=null]
                                Observation = [(DOUBLE_ARRAY)totalTIS_nesw_incoming=null, (DOUBLE_ARRAY)totalTIS_nesw_outgoing=null, (DOUBLE)totalTIS_intersection=null, (INTEGER)phaseIndex=null, (DOUBLE)time_mins=null]
                                Action      = [(INTEGER)do_next_phase=null]
[2021-12-01 19:39:52] WARNING @ org.glassfish.jersey.server.wadl.WadlFeature configure 
                              | JAX-B API not found . WADL feature is disabled.
[2021-12-01 19:39:52] INFO    @ org.glassfish.grizzly.http.server.NetworkListener start 
                              | Started listener bound to [localhost:51150]
[2021-12-01 19:39:52] INFO    @ org.glassfish.grizzly.http.server.HttpServer start 
                              | [HttpServer] Started.
```
No other line is written, because the code is stuck at this point. I also tried to change the port, but nothing changed

**Comment by t-wolfeadam** [comment]
This may be resolved in v1.0.0. I suggest upgrading and trying, and feel free to open a new issue if it remains.
